packages/components/src/components/gridbar/URLBox.tsx

```diff
@@ -151,7 +152,16 @@ export class URLBox extends React.Component<URLBoxProps, URLBoxState> {
         // setup known URL actions
         for (let i=0; i < actionsProp.length; i++) {
             if (actionsProp[i].toLowerCase() in urlActions) {
-                let urlAction = urlActions[actionsProp[i].toLowerCase()];
+                const actionName = actionsProp[i].toLowerCase();
+                if (actionName === ViewAction.NAME && queryInfo )
```

Comment:
It seems like we might want to gate on queryModel.showViewSelector as well here.  It could happen that a project has views defined but since we don't show the view selection for sample manager, including view here would be odd.